libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -531,8 +531,8 @@ void AC_PosControl::set_pos_target(const Vector3f& position)
 {
     _pos_target = position;
 
-    _flags.use_desvel_ff_z = false;
```

Comment:
This should be done in a separate commit.
Please follow : http://ardupilot.org/dev/docs/contributing.html